metabrainz/admin/__init__.py

```diff
@@ -8,7 +8,7 @@ class AuthMixin(object):
     """All admin views that shouldn't be available to public, must subclass this."""
 
     def is_accessible(self):
-        return current_user.is_authenticated() and current_user.musicbrainz_id in current_app.config['ADMINS']
```

Comment:
In one of the last versions of Flask-Login `is_authenticated` and some other methods became properties.